decks/react/02-props.mdx

```diff
@@ -0,0 +1,79 @@
+import { FullScreenCode } from "mdx-deck/layouts";
+
+## Props
+
+- Data input from other components within the application
+- The central mechanism by which components communicate with each other
+- `props` is the name of the argument the component will receive
+- Props are used to create dynamic behaviour
+
+---
+
+export default FullScreenCode
+
+```javascript Props Example
```

Comment:
wondering if it would be useful to also give an example like

```js
const Greeting = { name } => <h1>Hello, {name}!</h1>
``` 

Loops back into the destructuring of function arguments, and a bit more clear that `<Greeting name=` ... matches up to what is in the props